frontend/gist-frontend/src/gist-login/LoginOptionsComponent.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import UserLoginComponent from "./UserLoginComponent";
+import CreateUserComponent from "./CreateUserComponent";
+
+import { Box, Tab, Tabs, Heading } from "grommet";
+
+interface ILoginOptionsComponentProps {}
+
+interface ILoginOptionsComponentState {
+  isLoginTabActive: boolean;
+}
+
+export default class LoginOptionsComponent extends React.Component<
+  ILoginOptionsComponentProps,
+  ILoginOptionsComponentState
+> {
+  state = {
+    isLoginTabActive: true
+  };
+
+  onTabChange = () => {
+    this.setState({
+      isLoginTabActive: !this.state.isLoginTabActive
+    });
+  };
+
+  getLoginTitle = () => {
+    if (this.state.isLoginTabActive) {
+      return "login";
+    } else {
+      return "create";
+    }
+  };
+
+  render() {
+    let loginTitle: String = this.getLoginTitle();
```

Comment:
could do : 

`const {isLoginTabActive} = this.state`
`const title : string = isLoginTabActive? 'login' : 'create' `

to get rid of getLoginTitle() function

Reply:
did, done, smash